tensorboard/webapp/runs/store/utils.ts

```diff
@@ -81,3 +81,21 @@ export function groupRuns(
   }
   return runGroup;
 }
+
+/**
+ * Util function for composing `GroupBy` key and regex string.
+ */
+export function createGroupBy(
```

Comment:
Sorry for the trouble but can you write a small test that tests this utility?

Reply:
good call. added them.
except for not sure how to test `default:` any suggestions? I think it should be covered by the first two cases?